src/audio/drc/drc_math_lib/asin/matlab_fixpt/drc_asin_wrapper_fixpt.m

```diff
@@ -0,0 +1,13 @@
+function y = drc_asin_wrapper_fixpt(x)
```

Comment:
what are units of x?

Reply:
x_in = fi( **x**, 1, 2, 0, fm );
**x** unit is represented by input values of -1 & 1
From **init_struc_fixpt.m** you  can see x units
In x = fi(**[-1,1]**, 1, 2, 0, fm); 
Representation expansion is 1 Signed bit,  2 Word Length and 0 Fraction Length along with
fm = fimath('RoundingMethod', 'Convergent',...
	     'OverflowAction', 'Wrap',...
	     'ProductMode','FullPrecision',...
	     'SumMode','FullPrecision');

Reply:
please add as a comment explaining